luigi_pipeline/seqr_loading.py

```diff
@@ -36,58 +41,63 @@ def run(self):
         mt = hl.split_multi(mt)
         mt = HailMatrixTableTask.run_vep(mt, self.genome_version, self.vep_runner)
 
-        mt = SeqrVariantSchema(mt).annotate_all(overwrite=True).select_annotated_mt()
+        ref_data = hl.read_table(self.reference_ht_path)
+        clinvar = hl.read_table(self.clinvar_ht_path)
+        hgmd = hl.read_table(self.hgmd_ht_path)
+
+        mt = SeqrVariantSchema(mt, ref_data=ref_data, clinvar_data=clinvar, hgmd_data=hgmd).annotate_all(
+            overwrite=True).select_annotated_mt()
 
         mt.describe()
         mt.write(self.output().path)
 
-    @staticmethod
-    def validate_mt(mt, genome_version, sample_type):
-        """
-        Validate the mt by checking against a list of common coding and non-coding variants given its
-        genome version. This validates genome_version, variants, and the reported sample type.
-
-        :param mt: mt to validate
-        :param genome_version: reference genome version
-        :param sample_type: WGS or WES
-        :return: True or Exception
-        """
-        sample_type_stats = HailMatrixTableTask.sample_type_stats(mt, genome_version)
-
-        for name, stat in sample_type_stats.items():
-            logger.info('Table contains %i out of %i common %s variants.' %
-                        (stat['matched_count'], stat['total_count'], name))
-
-        has_coding = sample_type_stats['coding']['match']
-        has_noncoding = sample_type_stats['noncoding']['match']
+@staticmethod
+def validate_mt(mt, genome_version, sample_type):
```

Comment:
Why is this unindented? Looks like it's removed from the class, I don't think `self.validate_mt` above would work, and makes sense why it wouldn't fail since we disable validation?

Reply:
Sorry, this was a mistake. Fixed.